### ISSUE TITLE
is this related to issue

### ISSUE THREAD

**Issue by sideffect0** [OpenGenus/ProjectBot#1]
Is this the related to issue https://github.com/OpenGenus/Find.the.Lost/issues/3 ?

**Comment by AdiChat** [comment]
Yes, it is. Presently, we are deciding the technology stack for our application. The rest of the work flow will flow and will be divided among team members. If you are enthusiastic to take up a major role in building the application, please mention your skills in the issue named Technology Stack and member skills. #2 Hoping to get to board together.
